community/event-checklist.md

```diff
@@ -0,0 +1,95 @@
+
+✔️ Speakers of the event should have enough practical experience with the topic. The experience is necessary so that speaker can convey the content in the simplest way and clear doubts.
+
+✔️ It is highly recommended that the Slides for the event are made from scratch by the speaker.  Preparing from scratch will add creativity and newness to the content. New and simpler ways to present a topic might come out.  Also...
+
+​      🗸 Feel free to look up at previous slides at [kossiitkgp/slides](https://github.com/kossiitkgp/slides) repo for inspiration.
+
+​      🗸 Make sure that something important that has been covered in the past doesn't gets removed.
+
+​      🗸 Feel free to remove or edit stuff if you ever feel some part of the content is unnecessary and irrelevant, after discussing with team mates. 
+
+✔️ Make sure to take feedback on the slides(content) from other members of KOSS. It is suggested to have the discussion over a meet to discuss things like potential queries from audience, thinking of ways to simplify content.
+
+✔️ Refer [kossiitkgp/mistakes](https://github.com/kossiitkgp/mistakes) repository to get an idea of what sought of things can go wrong.
+
+✔️ It is a good idea to record the event(offline/online) for the sake of posterity.
+
+✔️ Make a Github repo for the workshop with name as event name and year which contains - installation instructions, slides, references, recording and others. (For example refer - [1](https://github.com/kossiitkgp/git-and-github-workshop-2020), [2](https://github.com/kossiitkgp/REST-APIs-in-Flask-Workshop-2021), [3](https://github.com/kossiitkgp/Linux-Shell-Vim-Workshop-2021))
+
+## 1.1 Internal
+
+- Keep in touch regularly with other team mates on updates regarding the event. Regular communication on #events channel is suggested.
+- Meet regularly to discuss stuff. It is advisable to send mail invites even for a smaller meet, to avoid procrastination.
+
+- **Making the workshop interactive**: It is always a good thing to keep the workshop interactive. If the workshop is conducted offline, we can have members of KOSS help people personally with hands on during the workshop and clear doubts more personally. But during an online workshop, this is not possible. So for sake of interaction, the speakers can encourage the audience to share the screen to show the work.  This acts as validation and also helps the rest of the audience. Asking the audience to share their screen in case of difficulties should also be encouraged. However note that,**do not force the audience to share their screen for sake of interaction. It has be voluntary and audience should not feel uncomfortable through such actions.**
+
+# 2. Technicalities - Workshop content
+
+## 2.1 How to decide the Topic for the workshop
+
+- Make sure that the topic adds value to the community. The topic doesn't have to be trendy or in vogue.
+- Atleast a couple of members at KOSS should have enough practical experience to speak about the topic
```

Comment:
This is true, but the main thing I wanted to convey was that people who are presenting should have enough practical experience with the topic, no matter how suitable the topic is for the current needs of people.

An anecdote: In KOSS-18, we were planning to conduct a golang workshop, whilst we were writing KWoC backend in golang. But we dropped the idea, because none of us had experience of at least one project in golang. We felt, we were not enough experienced to teach to an audience.
Obviously, one can learn the basics and deliver the content. But that will not be effective. It will be as if, we were just reproducing the content. We might face trouble in answering the queries of the audience. In all our workshops, we use our experience and try to simplify topics we faced difficulties while learning(like Git - staging area, sending PR, branching, REST API).
Hence I felt the criteria that to deliver a workshop, practical experience is necessary.

Reply:
@rakaar Practical experience with the topic before delivering a workshop is absolutely necessary.

Reply:
@pranitbauva1997 With the obsession of making this doc shorter, I didn't explain this point in detail(and at other places too). But sometimes, it is important to mention "why" when it is not straightforward, otherwise this doc would be like a set of doctrines. 
I added an explanation to this. If there are any other places, u find an explanation is required, like here it was, plz mention them.